### DISCUSSION TITLE
Change all event types to ft.Event.xxx to make it easier to find.

### DISCUSSION THREAD

**Discussion by uuleejie** [flet-dev/flet#3663] [Ideas]
It is recommended to change all event types to ft.Event.xxx to make it easier to find. e.g. **def route_change(e:ft. RouteChangeEvent):** can be changed to **def route_change(e:ft. Event.RouteChange)**。 Of course, I'm just a noob and don't know if this suggestion is stupid, and if it is, ignore it.

**Comment by ndonkoHenri** [comment]
Your suggestion is not bad... I get your point.
But why do you think atm they are "difficult" to find? In modern IDEs like pycharm and vscode, just by typing `ft.Event`, you will see all of them:
![image](https://github.com/user-attachments/assets/8a5eded0-ae38-4300-a50f-3087a31faa98)


Also the typing of events has been enhanced, so you can easily know of what type an event is:
 
<img width="850" alt="Bildschirmfoto 2024-07-18 um 13 44 33" src="https://github.com/user-attachments/assets/1391585b-f0c4-49c9-8e5c-aca65f30527f">

**Reply by uuleejie** [reply]
I'm using vscode, and as you said, it's also convenient that way. I learned a new skill, thank you!
